### PR TITLE
[zh]Add windows-storage.md chinese version

### PR DESCRIPTION
**Pull request by zhangxyjlu**
Work for https://github.com/kubernetes/website/issues/34033 win-4
add windows-storage.md chinese version

### CODE REVIEW

File: content/zh/docs/concepts/storage/windows-storage.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+<!--
+## Persistent storage {#storage}
+
+Windows has a layered filesystem driver to mount container layers and create a copy
+filesystem based on NTFS. All file paths in the container are resolved only within
+the context of that container.
+-->
+## 持久存储 {#storage}
+Windows 有一个分层文件系统驱动程序用来挂载容器层和创建基于 NTFS 的拷贝文件系统。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/windows-storage.md:32]:
```suggestion
Windows 有一个分层文件系统驱动程序用来挂载容器层和创建基于 NTFS 的文件系统拷贝。
```

File: content/zh/docs/concepts/storage/windows-storage.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+applications, with data persistence and Pod volume sharing requirements, to be deployed
+on Kubernetes. Management of persistent volumes associated with a specific storage
+back-end or protocol includes actions such as provisioning/de-provisioning/resizing
+of volumes, attaching/detaching a volume to/from a Kubernetes node and
+mounting/dismounting a volume to/from individual containers in a pod that needs to
+persist data.
+-->
+使用 Kubernetes {{< glossary_tooltip text="卷" term_id="volume" >}}，
+对数据持久性和 Pod 卷共享有需求的复杂应用也可以部署到 Kubernetes 上。
+管理与特定存储后端或协议相关的持久卷时，相关的操作包括：对卷的配备（Provisioning）、
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/windows-storage.md:91]:
```suggestion
管理与特定存储后端或协议相关的持久卷时，相关的操作包括：对卷的制备（Provisioning）、
```

File: content/zh/docs/concepts/storage/windows-storage.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+去配（De-provisioning）和调整大小，将卷挂载到 Kubernetes 节点或从节点上解除挂载，
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/windows-storage.md:92]: attach - 挂接、detach - 解除挂接、mount - 挂载、unmount - 解除挂载

```suggestion
去配（De-provisioning）和调整大小，将卷挂接到 Kubernetes 节点或从节点上解除挂接，
```

File: content/zh/docs/concepts/storage/windows-storage.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+将卷挂载到需要持久数据的 Pod 中的某容器上或从容器上卸载。
+
+<!--
+Volume management components are shipped as Kubernetes volume
+[plugin](/docs/concepts/storage/volumes/#types-of-volumes).
+The following broad classes of Kubernetes volume plugins are supported on Windows:
+-->
+卷管理组件作为 Kubernetes 卷[插件](/docs/concepts/storage/volumes/#types-of-volumes)发布。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/windows-storage.md:100]:
```suggestion
卷管理组件作为 Kubernetes 卷[插件](/zh/docs/concepts/storage/volumes/#types-of-volumes)发布。
```

File: content/zh/docs/concepts/storage/windows-storage.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+* [`FlexVolume plugins`](/zh/docs/concepts/storage/volumes/#flexVolume)
+  * 请注意 自 1.23 版本起，FlexVolume 已被弃用
+* [`CSI Plugins`](/zh/docs/concepts/storage/volumes/#csi)
+
+<!--
+##### In-tree volume plugins
+
+The following in-tree plugins support persistent storage on Windows nodes:
+-->
+#####  In-tree 卷插件
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/windows-storage.md:117]:
```suggestion
##### 树内（In-Tree）卷插件   {#in-tree-volume-plugins}
```

File: content/zh/docs/concepts/storage/windows-storage.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+
+以下 in-tree 插件支持 Windows 节点上的持久存储：
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/windows-storage.md:119]:
```suggestion
以下树内（In-Tree）插件支持 Windows 节点上的持久存储：
```

**Comment by zhangxyjlu** [comment]
I've revised all of your reviews，thank you@[tengqm](https://github.com/tengqm)

**Comment by Sea-n** [comment]
The format and [Netlify preview](https://deploy-preview-34136--kubernetes-io-main-staging.netlify.app/zh/docs/concepts/storage/windows-storage) looks good to me. Thanks!
/lgtm

**Pull request merged**